x-pack/legacy/plugins/security/public/views/login/login.tsx

```diff
@@ -20,6 +20,9 @@ const messageMap = {
   SESSION_EXPIRED: i18n.translate('xpack.security.login.sessionExpiredDescription', {
     defaultMessage: 'Your session has timed out. Please log in again.',
   }),
+  SESSION_ENDED: i18n.translate('xpack.security.login.sessionEndedDescription', {
+    defaultMessage: 'Your session has exceeded the maximum time limit. Please log in again.',
```

Comment:
@gchaps This message is only currently displayed after the user has been logged out and redirected to the login page.

Aside: we may want to add it to the "logged out" page in the future, though (which is only used for other auth realms like SAML). I believe we'll have to figure out another way to pass the message if we want to do that, because of the way that those auth realms / identity providers handle redirects.

Reply:
@jportner Thanks for the clarification.  The original message says it succinctly;

Your session has expired.  Please log in again.

Reply:
@gchaps after talking to Larry, we both agree it would probably just be simpler to go with the original message for both cases, whether the session timed out due to being idle or due to reaching its maximum lifespan.

'Your session has timed out. Please log in again.'

So I will just remove the additional message and leave this one as-is, unless you think we should change this to "has expired".